src/main/kotlin/com/wire/apps/polls/setup/KtorInstallation.kt

```diff
@@ -105,26 +98,6 @@ fun Application.installFrameworks() {
 
     install(DefaultHeaders)
 
-    install(CallLogging) {
-        // insert nginx id to MDC
-        mdc(INFRA_REQUEST) {
-            it.request.header("X-Request-Id")
-        }
-
-        // use generated call id and insert it to MDC
-        mdc(APP_REQUEST) {
-            it.callId
-        }
-
-        // enable logging just for /messages
-        // this filter does not influence MDC
-        filter {
-            it.request.uri == "/messages"
-        }
-        level = Level.DEBUG
-        logger = createLogger("HttpCallLogger")
-    }
-
     install(CallId) {
```

Comment:
I think you can also remove the callId plugin, it's related to server call tracing